src/rp2_common/pico_cyw43_driver/CMakeLists.txt

```diff
@@ -92,12 +92,13 @@ if (EXISTS ${PICO_CYW43_DRIVER_PATH}/${CYW43_DRIVER_TEST_FILE})
     # Set an ip address in a compile definition
     # target name, target type, compile definition name to set then address in a string
     # This can be used to set the following compile definitions
-    # CYW43_DEFAULT_IP_ADDRESS
+    # CYW43_DEFAULT_IP_STA_ADDRESS
     # CYW43_DEFAULT_IP_MASK
-    # CYW43_DEFAULT_IP_GATEWAY
+    # CYW43_DEFAULT_IP_STA_GATEWAY
```

Comment:
Minor nit-pick, but I guess this might look nicer if the `CYW43_DEFAULT_IP_STA_ADDRESS` and `CYW43_DEFAULT_IP_STA_GATEWAY` lines are adjacent? (just like `CYW43_DEFAULT_IP_AP_ADDRESS` and `CYW43_DEFAULT_IP_AP_GATEWAY` are adjacent)

Reply:
I agree. I didn't originally as I wanted to keep my changes as small as possible but yes, grouping them is worthwhile! I'll modify it.